src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -133,9 +133,16 @@ internal static ProcessInfo CreateProcessInfo(Interop.procfs.ParsedStat procFsSt
                 ProcessId = pid,
                 ProcessName = procFsStat.comm,
                 BasePriority = (int)procFsStat.nice,
-                VirtualBytes = (long)procFsStat.vsize,
-                WorkingSet = procFsStat.rss * Environment.SystemPageSize,
                 SessionId = procFsStat.session,
+                PoolPagedBytes = (long)procFsStatus.vmswap,
+                VirtualBytes = (long)procFsStatus.vmsize,
```

Comment:
should this match the old `(long)procFsStat.vsize` ? If so is it worth an assert? I guess not because the two reads are not atomically made.

Reply:
These could probably match but I didn't check precisely, and ike you are pointing out, there is a risk of races since both files are not read atomically.